openvdb_cmd/vdb_tool/examples/EXAMPLES.md

```diff
@@ -0,0 +1,61 @@
+# Documentation
+All the configuration files in this directory can be executed as
+```
+./surfacer -read points_to_mesh.txt
+```
+
+# Examples
+Print documentation to the terminal (and terminate)
+```
+./surfacer --help
+```
+
+Convert a polygon mesh file into to another polygon mesh file
+```
+./surfacer --read mesh.[obj|ply|stl] --write mesh.[obj|ply|stl]
+```
+
+Convert multiple obj files (mesh_{00-09}.obj) into multiple ply files (mesh_{00-09}.ply)
+```
+./surfacer --for f=0,10,1 --read mesh_{$f:2:pad0}.obj --write mesh_{$f:2:pad0}.ply -end
+```
+
+Convert a polygon mesh file into a narrow-band level and save it to a file
+```
+./surfacer --read mesh.obj -mesh2ls --write mesh.vdb
+```
+
+Convert a polygon mesh file into a narrow-band level of width 2 with maximum voxels dimension of 512 and save it to a file
+```
+./surfacer --read mesh.obj --mesh2ls dim=512 width=2 --write mesh.vdb
+```
+
+Converts input points in the file points.[ply|obj|stl|pts] to a level set, perform level set operations, and written to it the file surface.vdb
+```
+./surfacer --read points.[obj|ply|stl|pts] --points2ls --dilate --gauss --erode --write surface.vdb
+```
+
+Converts input points in the file points.vdb to a level set, perform level set operations, and written to it the file surface.vdb
+```
+./surfacer -read points.vdb -vdb2points -points2ls -dilate -gauss -erode -write surface.vdb
+```
+
+Converts input points in the file points.[ply|obj|stl|pts] to a level set, perform level set operations, extract a polygon mesh, and save the mesh to the file surface.[ply|obj|stl|pts]
+```
+./surfacer -read points.[ply|obj|stl|pts] -points2ls -dilate -gauss -erode -ls2mesh -write surface.[ply|obj|stl|pts]
+```
+
+This examples is more verbose and demonstrates how parameters of the level set operations are specified.Note that the dilation operation is using 5'order (WENO) spatial discretization and 2'nd order (TVD-RK) time discretization.
```

Comment:
Space and wording: "specified. Note that the dilation operation is using 5'th order"